CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 2.15.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.0.2210**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. New data sources:
+  - **data_source_aviatrix_firewall**
+  - **data_source_vpc_tracker**
+2. Implemented support for option to manage attachment on either **aviatrix_vpn_profile** or **aviatrix_vpn_user**
+3. Implemented support for ``action`` under domain_names for **aviatrix_fqdn**
+4. Implemented support for adding new users with Geo VPN
+5. Implemented support for ``ha_peering_subnet`` option for GCP provider for **aviatrix_gateway**
+6. Implemented support for ``ha_subnet`` option for GCP provider for **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+7. Implemented support for ``enable_ikev2`` for **aviatrix_site2cloud**
+
+### Enhancements:
+1. Updated ``tunnel_type`` option to be "policy" and "route"
+2. Added ``route_tables`` and ``route_tables_filter`` in **data_source_vpc**
+3. Updated **aviatrix_vpc** to return parsed vpc_id for GCP provider
+4. Updated terraform to cover the case that encrypt volume is enabled with gateway creation
```

Comment:
4. Updated provider to support unencrypted gateway volumes as an option for backward compatibility between existing and new gateways in Controller 6.0; new gateway volumes are encrypted by default on Controller.